nix/frontend.nix

```diff
@@ -34,6 +34,7 @@ stdenv.mkDerivation (finalAttrs: {
   pnpmDeps = pnpm.fetchDeps {
     inherit (finalAttrs) pname version src;
     fetcherVersion = 2;
+    pnpmInstallFlags = [ "--package-import-method=hardlink" ]; # Fixes 
```

Comment:
Does ``package-import-method=clone-or-copy`` work for you, too? I think me and marcel have that set locally because we also use ZFS.

Also the hook sets that in unstable https://github.com/NixOS/nixpkgs/blob/0cadc7de94c376c80819b4fb412078ecf5924ed3/pkgs/development/tools/pnpm/fetch-deps/pnpm-config-hook.sh#L43

Reply:
I do not have that locally, but it fails for me sometimes but not always. But until now it just always worked with retiries. In nix builds the error never occurred , my assumption was that it worked because it wasn't on zfs but on a tempfs.

Reply:
Looking at the bug reports it's actually `ref-clones` that cause the issue in the first place.